readme.md

```diff
@@ -1424,13 +1424,20 @@ your `vite.config.js`.
 
 #### WMR
 
-~~[WMR](https://github.com/preactjs/wmr) supports [Rollup][] plugins directly by
-[pushing them to `config.plugins`](https://github.com/preactjs/wmr/tree/main/packages/wmr#configuration-and-plugins)
-in `wmr.config.js`.~~
+[WMR](https://github.com/preactjs/wmr) supports [Rollup][] plugins directly by
+[adding them to `plugins`](https://wmr.dev/docs/plugins)
```

Comment:
We recently made our docs site public and moved much of the information out of the project's ReadMe, hence the link change. With that we've also made the decision to promote a different config API that gives better intellisense, so while pushing to `config.plugins` still works, that's not the API format we often show.

Example:
```
// wmr.config.mjs
import { defineConfig } from 'wmr';
import { MyExamplePlugin } from './my-example-plugin';

export default defineConfig({
    plugins: [MyExamplePlugin()]
});
```